{"name": "test_add[xfail]", "status": "skipped", "statusDetails": {"message": "AssertionError: assert 15 == 50", "trace": "self = <test_mycalc.Test_mycalc_Add object at 0x000001F5858F2D90>\nget_calc = <python_practice.python_commit_4.mycalc.myCalc object at 0x000001F585896D30>, a = 7, b = 8, expect = 50\n\n    @pytest.mark.add1\n    @pytest.mark.parametrize(\n        'a,b,expect',\n        [(1, 2, 3), (0.1, 0.8, 0.9), (0, 8, 8), pytest.param(7, 8, 50, marks=pytest.mark.xfail)],\n        ids=['int', 'float', 'zero', 'xfail']\n    )\n    @pytest.mark.run(order=5)\n    def test_add(self, get_calc, a, b, expect):\n        # calc_result = self.calc.add(a, b)\n        calc_result = get_calc.add(a, b)\n        if type(a) == float and type(b) == float:\n            a = round(a, 3)\n            b = round(b, 3)\n            calc_result = round(calc_result, 3)\n>       assert calc_result == expect\nE       assert 15 == 50\n\ntest_mycalc.py:108: AssertionError"}, "attachments": [{"name": "stdout", "source": "5dacd5bb-3a88-41ca-a7eb-6c6a3a196828-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "a", "value": "7"}, {"name": "b", "value": "8"}, {"name": "expect", "value": "50"}], "start": 1599305403524, "stop": 1599305403524, "uuid": "88f5d455-8c74-491c-ace9-cba9f52baeed", "historyId": "3bd5a520e9628c615c81fdf9055bf175", "testCaseId": "8489ccf5f663982291194cae28593af5", "fullName": "test_mycalc.Test_mycalc_Add#test_add", "labels": [{"name": "tag", "value": "run(order=5)"}, {"name": "tag", "value": "@pytest.mark.xfail"}, {"name": "tag", "value": "add1"}, {"name": "suite", "value": "test_mycalc"}, {"name": "subSuite", "value": "Test_mycalc_Add"}, {"name": "host", "value": "DESKTOP-4KTILRS"}, {"name": "thread", "value": "7788-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_mycalc"}]}